### PR TITLE
Backport: 6.0.x: flow manager: using too much CPU during idle 

### PR DESCRIPTION
**Pull request by jlucovsky**
Continuation of #7743 

Backport to 6.0.x -- flow manager

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: [4421](https://redmine.openinfosecfoundation.org/issues/4421)

Updates:
- Reduced scope of changes in support code for cherry-picked patches.

#suricata-verify-pr:
#suricata-verify-repo:
#suricata-verify-branch:
#suricata-update-pr:
#suricata-update-repo:
#suricata-update-branch:
#libhtp-pr:
#libhtp-repo:
#libhtp-branch:

### CODE REVIEW

File: src/flow-manager.c [hunk 1]
```diff
@@ -955,7 +1007,30 @@ static TmEcode FlowManager(ThreadVars *th_v, void *thread_data)
...
-        usleep(250);
+
+        if (emerg || !time_is_live) {
+            usleep(250);
+        } else {
+            struct timeval cond_tv;
+            gettimeofday(&cond_tv, NULL);
+            struct timeval add_tv;
+            add_tv.tv_sec = 0;
+            add_tv.tv_usec = (sleep_per_wu * 1000);
```
Comment (victorjulien) [inline src/flow-manager.c:1018]: we shouldn't backport the work unit stuff either, just the cond commits. They should sleep based on `next_run_ms` I suppose.

Comment (jlucovsky) [reply src/flow-manager.c:1018]: Ok ... Will rework and submit a new PR.

**Comment by suricata-qa** [comment]
WARNING:
 
 | field | test | baseline | % |
|--|--|--|--| 
| | generic_stats_chk | | |
| .capture.kernel_drops | 23548 | 3018 | 780.25% | 

Pipeline 8719
WARNING: THERE IS A KNOWN BAD BASELINE WITH PACKET DROPS. bE MINDFUL OF ANY RESULTS.

**Comment by jlucovsky** [comment]
Continued in #7757

**Pull request closed without merging**